app/assets/javascripts/bp_property_tree.js

```diff
@@ -301,6 +301,7 @@
           } else {
             ROOT.html("No properties exist for this ontology");
             ROOT.css("font-size", "14px").css("margin", "5px");
+            jQuery(".spinner-border").hide()
```

Comment:
this will hide all the spinners of the application, you need to select only the one that is the properties block

Reply:
![image](https://github.com/ontoportal-lirmm/bioportal_web_ui/assets/31127782/74061d3b-1ed8-44a4-a5df-3be07721cfb6)
In that case : here is the loader

Reply:
Yes but here the selector `.spinner-border` will select all the existent loaders in the site because it is a reused component, so they share the same class. 

The correct selector would be `#prop_contents .spinner-border` , the `#prop_contents` is the id of the container of the loader that you are showing me.

Reply:
Done, thank you for the solution